fipy/__init__.py

```diff
@@ -35,8 +35,37 @@
 from builtins import input
 __docformat__ = 'restructuredtext'
 
+import os
+
+# configure logging before doing anything else, otherwise we'll miss things
+if 'FIPY_LOG_CONFIG' in os.environ:
+    import json
```

Comment:
Is json a new dependency for FiPy? That's why it's inside the if statement

Reply:
It is a new requirement, but it's also in the standard library, so I guess there's no reason not to collect it with the imports at top